client/signup/steps/design-type/index.jsx

```diff
@@ -11,27 +13,39 @@ import SignupActions from 'lib/signup/actions';
 import analytics from 'lib/analytics';
```

Comment:
Since this is a janitorial PR, how about moving analytics to using [redux middleware](https://wpcalypso.wordpress.com/devdocs/client/state/analytics/README.md?term=state%20analytics) as well?